src/app/modules/create-facility/pages/review/review-applicant/review-applicant.component.ts

```diff
@@ -24,25 +25,23 @@ export class ReviewApplicantComponent implements OnInit {
     this.review.redirectPath = ROUTES_FACILITY.APPLICANT.fullpath;
     this.review.header = ROUTES_FACILITY.APPLICANT.title;
 
-    const form =RandomObjects.getApplicant(''); // this.updateStateService.forms.organizationForm;
+    const form =RandomObjects.getApplicant(''); 
```

Comment:
Individual page components should not generate randomized dummy data.   All randomized data generation should happen in a single service, and it should be controllable via the env variable "useDummyData".    This way we don't have to worry about dummy data accidentally slipping into production builds.

Currently I use `CreateFacilityDataService` to create all dummydata, but if you'd prefer we could have a CreateFacilityDummyDataService which serves the sole purpose of setting values in the data service.

Take a look at what I do in CreateFacilityDataService.  Right now I'm just doing everything in the constructor.   You should be able to delete RandomObjects and put all the dummy values directly in the CreateFacilityDataService.

Reply:
Another way of thinking about it -

When "useDummyData" is true, then the data in the data-service should be setup so that it looks identical to the user filling out the forms manually.  If you follow this pattern it winds up being a lot less work.

For example, you reference RnadomObjects in 5 different components.  This is a lot more boilerplate and more code to remove.  If you put all the dummy data directly on the service, then you only have to ever modify one place.

Reply:
Thanks  @acoard @adamcoard-maximus  for updating. I removed those ones.